CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.19)
+cmake_minimum_required(VERSION 3.18)
```

Comment:
Don't change this, `X11_xcb_xfixes_LIB` was added in 3.19.

Reply:
But i can't compile plugin without this fix. It compiles without cursor support, but without warnings and errors too.

Reply:
Alright then, it should then be lowered even more since the only reason the requirement is that high is for the xcb-xfixes.

Reply:
I can try to hide this change and leave choice to you.

Reply:
Also, this guy somehow make cursor support into this plugin:
https://github.com/w23/obs-kmsgrab/tree/obs-27-fixes

Reply:
It's just that FindX11.cmake that is shipped with CMake doesn't export xcb-xfixes library variable until 3.19.
Alternatively, FindX11.cmake can be just copied into the project, but I chose not to.